manifests/05-version-mapping.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: openshift-console-operator
```

Comment:
When we were discussing the need to `WATCH` this configmap, we contemplated if it needs to be in `openshift-console-operator`, which means we have to setup a bunch of new informers in our `starter.go`.  If we put it in `openshift-console`, we can just add an additional name to `tweakListOptionsForConfigs func`, which would be a lot less of a hassle.    

I'm not sure yet if we need to watch it, looking around.